src/main/webapp/dev/js/main/feedbackSubmissionsEdit.es6

```diff
@@ -963,10 +963,12 @@ $(document).ready(() => {
     if (typeof richTextEditorBuilder !== 'undefined') {
         $.each(textFields, (i, textField) => {
             const id = $(textField).attr('id');
+            const isSessionClosed = $(textField).data('is-session-closed') || false;
```

Comment:
> I think the variable name makes it clear enough about which side is receiving the benefit of the doubt either way.

@HirdayGupta `isSessionClosed = false` is a double negation and slightly less clear haha.